<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="uRLLabel">
     <property name="text">
      <string>URL</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtUrl"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="paramsLabel">
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="txtParams"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="layerLabel">
     <property name="text">
      <string>Layer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtLayer"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
